Makefile

```diff
@@ -7,7 +7,7 @@ build-rvsol:
 .PHONY: build-rvsol
 
 build-test:
-	make -C ./tests/go-tests all
+	make -C ./tests/go-tests bin bin/simple bin/minimal
```

Comment:
How about adding `build-bin` command and call it?

Reply:
added all-test command into the go-tests makefile.